main.py

```diff
@@ -71,6 +74,32 @@ def on_keypress(bit_index):
       alive_enemies.remove(enemy)
     else:
       enemy.update_position()
+      
+  if len(missiles_to_shoot) > 0:
+    for missiles in missiles_to_shoot:
+      if missiles.has_collided:
+        missiles.enemy.is_destroyed = True
+        missiles.enemy.update_position()
+        missiles_to_shoot.remove(missiles)
+        continue
+      if len(missiles.missiles_in_place) == len(missiles.missiles):
+        missiles.is_shot = True
```

Comment:
Once the condition is met, it continues to be met every tick, and so re-sets `is_shot` every tick.